cmd/kubeadm/app/apis/kubeadm/env.go

```diff
@@ -25,7 +25,7 @@ import (
 
 // TODO(phase2) use componentconfig
 // we need some params for testing etc, let's keep these hidden for now
-func GetEnvParams() map[string]string {
+func GetEnvParams(env map[string]string) map[string]string {
```

Comment:
I took a stab at creating a GlobalEnvParams, hopefully I was able to capture what you mean.  Please let me know!